From aaf5d1d57b9f6f5814c3f19680617b13a88a6f35 Mon Sep 17 00:00:00 2001
From: Fang Hui <b31070@freescale.com>
Date: Tue, 26 Nov 2013 15:32:30 +0800
Subject: [PATCH 261/281] ENGR00215174 update to pass build for android 4.4

Under user config, evk_6sl system directory is 278MB,
so 280MB size for ext4 system is not enough, enlarge to 360MB

Signed-off-by: Fang Hui <b31070@freescale.com>
---
 imx6/BoardConfigCommon.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index db45d85..4aa434f 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -58,7 +58,7 @@ ARCH_ARM_USE_NON_NEON_MEMCPY := true
 
 BOARD_RECOVERY_PARTITION_SIZE = 10M
 
-BOARD_SYSTEMIMAGE_PARTITION_SIZE := 293601280
+BOARD_SYSTEMIMAGE_PARTITION_SIZE := 377487360
 BOARD_FLASH_BLOCK_SIZE := 4096
 TARGET_RECOVERY_UI_LIB := librecovery_ui_imx
 
-- 
1.8.0

